src/cli.ts

```diff
@@ -8,10 +8,17 @@ function parseInputOptions(inputOptions: string[]): Input[] | undefined {
     const inputs: Input[] = [];
     inputOptions.map((inputOption) => {
       const [type, keyAndFilename] = inputOption.split(':');
```

Comment:
This will silently drop hlsName part.

```suggestion
      const [type, keyAndFilename, hlsName] = inputOption.split(':');
```